relayer/chains/evm/client.go

```diff
@@ -66,10 +54,8 @@ func newClient(ctx context.Context, connectionContract, XcallContract common.Add
 
 // grouped rpc api clients
 type Client struct {
-	log *zap.Logger
-	rpc *rpc.Client
-	eth *ethclient.Client
-	// evm chain ID
+	log        *zap.Logger
```

Comment:
I don't think zap logger is needed here. Basically logging is done in the core business logic.

Reply:
Can be removed entirely but removing struct have larger impact than having it, it's just a interface to pointer not any gains.